### PR TITLE
fix(Object): Fixes centeredScaling prop type

### PR DESCRIPTION
**Pull request by HesamSoleymani**
## Motivation

centeredScaling type is set to "false" instead of "boolean" in v6
As @ShaMan123  mentioned in [this discussion](https://github.com/fabricjs/fabric.js/discussions/9398), it looks like a mistake...

## Description

```
  /**
   * When true, this object will use center point as the origin of transformation
   * when being scaled via the controls.
   * <b>Backwards incompatibility note:</b> This property replaces "centerTransform" (Boolean).
   * @since 1.3.4
   * @type Boolean
   * @default
   */
  centeredScaling: false;
```

## Changes

I've just changed centeredScaling type in FabricObjectProps.ts

### CODE REVIEW

**Review by ShaMan123 — COMMENTED** [review]
thanks, could it be that it is optional as well?

File: src/shapes/Object/types/FabricObjectProps.ts [hunk 1]
```diff
@@ -28,7 +28,7 @@ export interface FabricObjectProps
    * @type Boolean
    * @default
    */
-  centeredScaling: false;
+  centeredScaling: boolean;
```
[the remark this answers is not in the document]
Comment (HesamSoleymani) [reply src/shapes/Object/types/FabricObjectProps.ts:31]: Is it necessary? I mean I think Partial<FabricObjectProps> is used everywhere so every option is optional...
If it is, then what about other props which are still not optional in FabricObjectProps ? (such as centeredRotation, selectionBackgroundColor, selectable and ...)

[the remark this answers is not in the document]
Comment (ShaMan123) [reply src/shapes/Object/types/FabricObjectProps.ts:31]: I will give a look and follow up if needed

**Comment by ShaMan123** [comment]
Can you add a log to the changlog?
Then I can merge

**Review by asturur — CHANGES_REQUESTED** [review]
@HesamSoleymani please also change this here:
https://github.com/fabricjs/fabric.js/blob/master/src/shapes/Object/InteractiveObject.ts#L58
to a declare boolean

**Comment by HesamSoleymani** [comment]
Everything is ok now about this pr?

**Review by asturur — DISMISSED** [review]

**Comment by asturur** [comment]
i think so.
@ShaMan123 wanted to have a look at something, i approved, when he has time he can check and merge or comment.

**Review by ShaMan123 — APPROVED** [review]

**Pull request merged**